graylog2-web-interface/src/views/components/contexts/SearchPagePreferencesProvider.test.tsx

```diff
@@ -86,29 +86,29 @@ describe('SearchPageLayoutProvider', () => {
     return consume;
   };
 
-  it('provides default search page layout with empty preference store', () => {
+  it('provides default search page preference state with empty preference store', () => {
     const consume = renderSUT();
 
     expect(consume.mock.calls[0][0]?.config.sidebar.isPinned).toEqual(false);
```

Comment:
Not related to your code, but maybe you have some time to fix it: These could be replaced by something like this:
```suggestion
    expect(consume).toHaveBeenCalledWith(expect.objectContaining({ config: expect.objectContaining({ sidebar: expect.objectContaining({ isPinned: false }) }) }));
```
This way we do not have to make assumptions about the mock object.